### PR TITLE
modify middleware settings and url paths for django 2.0

### PR DESCRIPTION
**Pull request by hirokts**
Please check the django 2.0 middleware settings  of the official document.

(pip requirements.txt:    Django==2.0.3)

https://docs.djangoproject.com/en/2.0/topics/http/middleware/

### CODE REVIEW

File: container_engine/django_tutorial/Dockerfile [hunk 1]
```diff
@@ -21,11 +21,11 @@ FROM gcr.io/google_appengine/python
 
 # Create a virtualenv for the application dependencies.
 # # If you want to use Python 3, add the -p python3.4 flag.
-RUN virtualenv /env
+RUN virtualenv -p python3.4 /env
```
Comment (sketchytechky) [inline container_engine/django_tutorial/Dockerfile]: I bumped this to 3.6 and confirmed it works 👌

Comment (sketchytechky) [inline container_engine/django_tutorial/Dockerfile]: I bumped this to 3.6 and confirmed it works 👌

Comment (hirokts) [reply container_engine/django_tutorial/Dockerfile]: Thanks. I changed the version.

File: container_engine/django_tutorial/Dockerfile [hunk 2]
```diff
@@ -21,11 +21,11 @@ FROM gcr.io/google_appengine/python
 
 # Create a virtualenv for the application dependencies.
 # # If you want to use Python 3, add the -p python3.4 flag.
```
Comment (andrewsg) [inline container_engine/django_tutorial/Dockerfile]: We should change this comment, since python36 is preselected.

Comment (hirokts) [reply container_engine/django_tutorial/Dockerfile]: Thanks, I changed it.

**Comment by hirokts** [comment]
I confirmed it's ok to do so.
The reason of this change is "middleware settings needs to be changed due to differences in version of django", I prefer declarative version specification. Although, assuming automatically updating the version of django (in pip requirements text) or python (in docker image),  it's better to use "-p python3" or "python3 -m venv".

**Comment by andrewsg** [comment]
We do often update versions of dependencies such as Django as part of our maintenance (via bots). Given that, should we change it to -p python3 or do you think it should stay as it is?

**Comment by morenoh149** [comment]
I fixed these issues myself. I also had to make the changes described here https://gist.github.com/morenoh149/39be1ace1f99b9571fd2b6b96098075d I could submit these as a separate PR but may make logical sense to include in this PR

**Comment by hirokts** [comment]
Thanks.
Thinking about the situation, I changed python version specification.

I think it's better to change url path settings for django 2.
https://docs.djangoproject.com/en/2.0/ref/urls/
But I cannot confirm that immediately.

**Comment by morenoh149** [comment]
lgtm

**Comment by hirokts** [comment]
@andrewsg ,  could you check this PR?　Editing my comment may have been wrong if you did not notice the PR changing.

**Comment by andrewsg** [comment]
lgtm. Thanks!

**Pull request merged**